src/Microsoft.ML.Featurizers/Common.cs

```diff
@@ -221,30 +224,25 @@ internal static TypeId GetNativeTypeIdFromType(this Type type)
             throw new InvalidOperationException($"Unsupported type {type}");
         }
 
-        // The Native Featurizers do not currently support CentOS7, this method checks the OS and returns true if it is CentOS7.
-        internal static bool OsIsCentOS7()
+        internal static string PointerToString(IntPtr data)
         {
```

Comment:
It appears we are walking the native C string to find the null pointer and the length. Is this a perf optimization? If so have we measured it?

Reply:
This is tied to the previous comment. Since we are now not returning the size we need to walk the c string to find the end. Not a perf optimization, just needed based on interop changes.